cli/pkg/console/console.go

```diff
@@ -1,20 +1,34 @@
 package console
 
 import (
+	"bufio"
 	"fmt"
+	log "log"
```

Comment:
I was curious why `log "log"` was duplicated here, particularly if it's a quirk to Go that modifies the behavior of `log` here. I follow that capital-letter declarations are public and lowercase are private, but not sure if that rule applies here (especially since the import and the alias are both lowercase). Not at all a big deal, but just wanted to learn more about what this declaration does.